app/controllers/system_admin/lead_schools/users_controller.rb

```diff
@@ -1,30 +0,0 @@
-# frozen_string_literal: true
-
-module SystemAdmin
-  module LeadSchools
-    class UsersController < ApplicationController
-      before_action :set_lead_school
-      before_action :set_user
-
-      def edit; end
-
-      def update
-        if @user.update(permitted_attributes(@user))
-          redirect_to(lead_school_path(@lead_school), flash: { success: t(".success") })
-        else
-          render(:edit)
-        end
-      end
-
-    private
-
-      def set_lead_school
-        @lead_school = policy_scope(School, policy_scope_class: LeadSchoolPolicy::Scope).find(params[:lead_school_id])
```

Comment:
Potentially the `LeadSchoolPolicy` could be removed as well if not used anywhere else.